src/core/symbology/qgsrulebasedrenderer.h

```diff
@@ -383,10 +383,11 @@ class CORE_EXPORT QgsRuleBasedRenderer : public QgsFeatureRenderer
          *
          * \param ruleElem  The XML rule element
          * \param symbolMap Symbol map
+         * \param reuseId TRUE to create a copy of the symbol, FALSE to create a disctinct rule. Default is TRUE
```

Comment:
```suggestion
         * \param reuseId set to TRUE to create an exact copy of the original symbol or FALSE to create a new rule with the same parameters as the original but a new unique ruleKey(). (Since QGIS 3.30)
```